docker/oso-saml-sso/centos7/start.sh

```diff
@@ -23,7 +24,9 @@ echo group:x:$(id -G | awk '{print $2}'):user >> /etc/group
     config_version_dir=$(readlink -f /secrets/..data)
     echo "Running config playbook"
     for attempt_number in {1..3}; do
-      if ansible-playbook /root/config.yml; then
+      rm -f "$reconf_touchfile"
+      # if the playbook is successful, it will touch $reconf_touchfile as its final task
```

Comment:
Suggestion: comment should indicate that ansible-playbook returns success even if it fails.